### ISSUE TITLE
Seekbar with TalkBack enabled is not focusable/controllable (not accessibility compliant)

### ISSUE THREAD

**Issue by greyski** [google/ExoPlayer#7573]
### [REQUIRED] Searched documentation and issues
I've searched around the repo & official ExoPlayer docs, and only found a StackOverflow QA which I wanted to discuss: https://stackoverflow.com/questions/41418291/exoplayer-seekbar-does-not-work-with-arrow-keys

### [REQUIRED] Question
Does ExoPlayer plan on supporting TalkBack for the seekbar?

If not, does the referenced StackOverflow post ring true that we should implement a layout using fastforward/rewind when talkback is enabled?

Is there already an XML layout that the ExoPlayer library provides which the project owners consider accessible (TalkBack) compliant?

If not, do you plan on including an accessible compliant layout for developers using the library?

### A full bug report captured from the device
No bug report per se, simply enable TalkBack and attempt to interact with the ExoPlayer seekbar

### Link to test content
N/A

**Comment by kim-vde** [comment]
Hi,

I made a quick test with TalkBack (without external keyboard). What I observed is that, in the ExoPlayer demo app, the seek bar is indeed not focusable, but you can seek into the media by using the fast forward and rewind buttons. I observe the same behaviour in the Youtube app.

Is this improvement about making the seek bar focusable and navigable when an external keyboard is connected to the phone (as in [here](https://support.google.com/accessibility/android/answer/6110948?hl=en))?

**Comment by greyski** [comment]
No external keyboard, you've pretty much answered my question. If I'm summing this up correctly: The seekbar is not accessible on purpose and there is no plans on making it accessible, we utilize the fast forward and rewind buttons to support accessibility in regards to supporting seek actions.

If that's incorrect, feel free to update. Otherwise you've been very helpful, thank you!
